src/components/Tabs/Tabs.js

```diff
@@ -30,22 +31,54 @@ export default class Tabs extends React.Component {
   // Render current active tab content
   renderActiveTabContent() {
     const { children } = this.props;
-    const { activeTabIndex } = this.state;
+    const { activeTabIndex, horizontalLineWidth } = this.state;
     if (children[activeTabIndex]) {
-      return children[activeTabIndex].props.children;
+
+      let newChildren = [];
+      for (let i = 0; i < children[activeTabIndex].props.children.length; i++) {
+        let child = children[activeTabIndex].props.children[i];
+        console.log(child);
+        if (child.props.className && child.props.className.includes("centered_elements")) {
+          let newChild = React.cloneElement(child, {
+            style: {
+              maxWidth: horizontalLineWidth
+            }
+          });
+          newChildren.push(newChild);
+        }
+
+        else {
+          newChildren.push(child);
+        }
+      }
+      return newChildren;
     }
   }
 
   render() {
     return (
       <div className="tabs">
-        <ul className='tabs__ul'>
+        <div className="tabs__ul">
+
           {this.renderChildrenWithTabsApiAsProps()}
-        </ul>
+        </div>
         <ul className="tabs-active-content">
           {this.renderActiveTabContent()}
         </ul>
       </div>
     );
   }
-};
+
+  updateLineWidth() {
+    let buttons = Array.from(document.getElementsByClassName("tab"));
+    let width = 0;
+    buttons.forEach(b => {
```

Comment:
This can be simplified with: 

let width = buttons.reduce((accu, b) => accu+=b.clientwidth, 0); 

The accu is the variable that gets modifed on each iteration, accumulating the values. In this case, it's summing up the b's clientwidth. The 0 is needed to initialize the accu. 

More details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Reply:
by doing that it sets the width to NaN(attached image from console)
![Screen Shot 2021-02-21 at 9 43 23 PM](https://user-images.githubusercontent.com/49081374/108650558-d48aa980-748d-11eb-83b1-a95b1f50690c.png)

Reply:
If you are getting a Nan then one of two things is happening:

1. The accumulator is not being initialized. The second parameter (the 0) should be initializing the accumulator to 0
2. One or more of the clientwidth is null/undefined.